### PR TITLE
Updated invitation.yml

### PR DESCRIPTION
**Pull request by Vrukshali-26**
closes #ISSUE-NO

### CODE REVIEW

**Review by vj-abishek — DISMISSED** [review]
LGTM

File: .github/workflows/invitation.yml [hunk 1]
```diff
@@ -7,11 +7,11 @@ jobs:
...
-        uses: vj-abigo/invite-on-label@v1.1
+        uses: vj-abigo/invite-on-label@v1.2
         with:
           organization: EddieJaoudeCommunity
           label: invite me to the organisation
           repo-token: ${{ secrets.GITHUB_TOKEN }}
           comment: '<b>Invitation sent for the GitHub Organisation. Welcome to the community 🎉</b><br><br>Don''t forget after accepting to make it public so it appears on your GitHub profile for everyone else to see, you can do this by finding your name in the GitHub organisation list and change the dropdown to public https://github.com/orgs/EddieJaoudeCommunity/people<br><br>Tips for practising:<br><br>Customise your GitHub profile, here is a live stream on it https://www.youtube.com/watch?v=cT6GXCuS0Zo<br>Practice repo, instructions on how to add your name to the README in the README https://github.com/EddieJaoudeCommunity/hacktoberfest-practice<br>Remember contributing to open source is not just about code, its about collaboration, communication and adding value<br>I hope that helps'
         env:
-          ACCESS_TOKEN: ${{ secrets.ACCESS_TOKEN }}
+          INVITE_TOKEN: ${{ secrets.ACCESS_TOKEN }}
```
Comment (eddiejaoude) [inline .github/workflows/invitation.yml:17]: the secret needs changing too from `secrets.ACCESS_TOKEN` to be `secrets.INVITE_TOKEN`

Comment (vj-abishek) [reply .github/workflows/invitation.yml:17]: You also need to change in repo secrets. Which means you need to create a new token. So only I said like that.

**Review by vj-abishek — APPROVED** [review]
LGTM

**Review by eddiejaoude — APPROVED** [review]
Thanks 👍

**Pull request merged**